packages/core/src/external/sftp/surescripts/schema/load.ts

```diff
@@ -0,0 +1,322 @@
+    key: "senderId",
+    toSurescripts: toSurescriptsString("senderId"),
+  },
+  {
+    field: 3,
+    key: "senderPassword",
+    toSurescripts: toSurescriptsString("senderPassword"),
+  },
+  {
+    field: 4,
+    key: "receiverId",
+    toSurescripts: toSurescriptsString("receiverId"),
+  },
+  {
+    field: 5,
+    key: "transmissionId",
+    toSurescripts: toSurescriptsString("transmissionId"),
+  },
+  {
+    field: 6,
+    toSurescripts: toSurescriptsDate("transmissionDate"),
+  },
+  {
+    field: 7,
+    toSurescripts: toSurescriptsTime("transmissionDate", { centisecond: true }),
+  },
+  {
+    field: 8,
+    key: "transmissionFileType",
+    toSurescripts: toSurescriptsEnum("transmissionFileType", ["PAT", "PNM", "PMA"]),
+  },
+  {
+    field: 9,
+    key: "transmissionAction",
+    toSurescripts: toSurescriptsEnum("transmissionAction", ["U"], { optional: true }),
+  },
+  {
+    field: 10,
+    key: "extractDate",
+    toSurescripts: toSurescriptsDate("extractDate", { optional: true }),
+  },
+  {
+    field: 11,
+    key: "usage",
+    toSurescripts: toSurescriptsEnum("usage", ["test", "production"]),
+  },
+  {
+    field: 12,
+    key: "patientPopulationId",
+    toSurescripts: toSurescriptsString("patientPopulationId"),
+  },
+  {
+    field: 13,
+    key: "fileSchedule",
+    toSurescripts: toSurescriptsEnum("fileSchedule", ["ADHOC"]),
+  },
+  {
+    field: 14,
+    key: "lookBackInMonths",
+    toSurescripts: toSurescriptsInteger("lookBackInMonths", { optional: true }),
+  },
+];
+
+// These are the subsequent rows (details) of a patient load operation.
+export const patientLoadDetailSchema = z.object({
+  recordType: z.enum(["PAT", "PNM", "PMA"]),
+  recordSequenceNumber: z.number(),
+  assigningAuthority: z.string().min(1).max(64),
+  patientId: z.string().min(1).max(35),
+  lastName: z.string().min(1).max(35),
+  firstName: z.string().min(1).max(35),
+  middleName: z.string().min(1).max(35).optional(),
+  prefix: z.string().min(1).max(10).optional(),
+  suffix: z.string().min(1).max(20).optional(),
+  addressLine1: z.string().min(1).max(55).optional(),
+  addressLine2: z.string().min(1).max(55).optional(),
+  city: z.string().min(2).max(30).optional(),
+  state: z.string().min(2).max(55).optional(),
+  zip: z.string().min(5).max(10),
+  dateOfBirth: z
+    .string()
+    .length(8)
+    .regex(/^\d{8}$/),
+  genderAtBirth: z.enum(["M", "F", "U"]),
+  npiNumber: z.string().min(1).max(10),
+  endMonitoringDate: z.date().optional(),
+  requestedNotifications: z
+    .array(
+      z.enum([
+        "PMANewRx",
+        "PMARefill",
+        "PMANewSubscriber",
+        "PMAControlledSubstance",
+        "PMANoRefillsRemaining",
+        "PMARefillNotPickedUp",
+        "PMAInfo",
+      ])
+    )
+    .optional(),
+  startDate: z.date().optional(),
+  endDate: z.date().optional(),
+  primaryPhone: z.string().max(10).optional(),
+});
+
+export const patientLoadDetailOrder: OutgoingFileRowSchema<PatientLoadDetail> = [
```

Comment:
@thomasyopes If you get a chance, could you quickly look over how I am managing schemas here to verify that this is usable + in the right direction?